src/main/java/no/nav/fo/veilarbregistrering/registrering/bruker/HentRegistreringService.java

```diff
@@ -23,7 +27,7 @@ public HentRegistreringService(
 
     public OrdinaerBrukerRegistrering hentOrdinaerBrukerRegistrering(Bruker bruker) {
         OrdinaerBrukerRegistrering ordinaerBrukerRegistrering = brukerRegistreringRepository
-                .hentOrdinaerBrukerregistreringForAktorId(bruker.getAktorId());
+                .hentOrdinaerBrukerregistreringForAktorIdOgTilstand(bruker.getAktorId(), Status.OVERFORT_ARENA, Status.PUBLISERT_KAFKA, Status.OPPRINNELIG_OPPRETTET_UTEN_TILSTAND);
```

Comment:
Kunne vi her ha hatt en metode på Status som gir oss disse 3 verdiene som en liste ut av boksen? Da kan også metodenavnet beskrive hva som returneres, og vi kan forenkle koden litt i repo.

Reply:
Når jeg tenker meg om, så er det kanskje fint om repo-metoden tar inn en liste med Statuser som her, men at en kan ha en metode for å utlede listen. Da kan vi bruke den samme logikken andre steder, og slipper å oppdatere mange steder manuelt ved evt. endringer.